src/olympia/reviewers/utils.py

```diff
@@ -66,6 +66,7 @@ class AddonQueueTable(tables.Table):
         accessor='_current_version__autoapprovalsummary__score',
     )
     show_count_in_dashboard = True
+    view = None
```

Comment:
Can we define `view = queue` directly here? Or do we come up against circular reference errors?

Reply:
importing the functions from views leads to a total crash so I assume it's circular reference.

Reply:
idea: replace `views.reviewer_tables_registry[queue].view` in urls.py with `getattr(views, reviewer_tables_registry[queue].view_name)()` and change `view` in the table classes to be a string called `view_name`.

(other hacks may be available!)